app/package.json

```diff
@@ -39,12 +39,14 @@
   "jest": {
     "moduleNameMapper": {
       "^axios$": "axios/dist/axios.js"
-    }
+    },
+    "coveragePathIgnorePatterns": ["/node_modules/", "tests"]
```

Comment:
Should this be `src/tests`? or is `coveragePathIgnorePatterns` sensitive to /s?

Reply:
it's a regex so this will match "tests" anywhere in the path